LocalFeeder/requirements.txt

```diff
@@ -11,5 +11,5 @@ boto3
 xarray
 fastapi
 uvicorn
-git+https://github.com/openEDI/GADAL
+git+https://github.com/openEDI/GADAL@al/kubernetes
```

Comment:
Why is this needed now that it's on main?

Reply:
because testing requires it

Reply:
I'll leave this be for now.